content/docs/command-reference/version.md

```diff
@@ -12,20 +12,18 @@ usage: dvc version [-h] [-q | -v]
-| `Package`                                   | Name of the package manager used to install DVC if any (`pip`, `conda`, etc)                                                                                          |
-| `Supported remotes`                         | Remote types that have all needed dependencies installed for                                                                                                          |
-| `Cache`                                     | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported between the <abbr>workspace</abbr> and the <abbr>cache</abbr> |
-| `Filesystem type`                           | Shows the filesystem type (eg. ext4, FAT, etc.) and mount point of the cache and <abbr>workspace</abbr> directories                                                   |
-| `Repo`                                      | Shows whether we are in a DVC repo and/or Git repo                                                                                                                    |
-
-> If `dvc version` is executed outside a DVC project, no `Cache` info is
-> printed, and the `Filesystem type` output is of the current working directory.
+| Line                                        | Detail                                                                                                                                                               |
+| ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC, and name of the binary or package manager (`pip`, `conda`, etc.) used to install DVC (along with a Git commit hash in case of a development version) |
+| `Platform`                                  | Version of Python in the environment where DVC is initialized, and information about the operating system                                                            |
+| `Supports`                                  | Type of [remote storages](/doc/command-reference/remote/add#supported-storage-types) that are supported (their required dependencies are installed)                  |
+| `Cache types`                               | [Type of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) supported (between <abbr>workspace</abbr> and <abbr>cache</abbr>)      |
+| `Cache directory`                           | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>cache</abbr> directory is mounted                                                               |
+| `Workspace directory`                       | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>workspace</abbr> directory is mounted                                                           |
```

Comment:
The term "workspace directory" is redundant, especially in `<abbr>` because its defined as a "directory". Removing:
```suggestion
| `Workspace directory`                       | Filesystem type (e.g. ext4, FAT, etc.) and drive on which the <abbr>workspace</abbr> is mounted                                                           |
```